src/api_manager/fetch_metadata.cc

```diff
@@ -132,8 +158,25 @@ void GlobalFetchServiceAccountToken(
 void FetchServiceAccountToken(
     std::shared_ptr<context::RequestContext> request_context,
     std::function<void(utils::Status)> on_done) {
+  std::string audience;
+  GlobalFetchServiceAccountToken(
+      request_context->service_context()->global_context(), audience, on_done);
+}
+
+// Fetches instance identity token from metadata server.
+void FetchInstanceIdentityToken(
+    std::shared_ptr<context::RequestContext> request_context,
+    std::function<void(utils::Status)> on_done) {
+  std::string audience;
+  if (request_context->method()) {
+    audience = request_context->method()->backend_jwt_audience();
```

Comment:
It is better not to copy the audience string here

Reply:
Done.